jira-ruby.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   # Rubtime Dependencies
   s.add_runtime_dependency 'oauth', '~> 0.5', '>= 0.5.0'
-  s.add_runtime_dependency 'activesupport', '~> 4.2', '>= 4.2.0'
+  s.add_runtime_dependency 'activesupport'
 
   # Development Dependencies
   s.add_development_dependency 'railties', '~> 4.2', '>= 4.2.0'
```

Comment:
Is a version restriction on this dependency needed? Railties is also up at version 5.0-stable for Rails 5 support.